components/logins/src/db.rs

```diff
@@ -134,6 +185,19 @@ impl LoginDb {
     }
 }
 
+fn sqlcipher_3_compat(conn: &Connection) -> Result<()> {
+    // SQLcipher pre-4.0.0 compatibility. Using SHA1 still
+    // is less than ideal, but should be fine. Real uses of
+    // this (lockwise, etc) use a real random string for the
+    // encryption key, so the reduced KDF iteration count
+    // is fine.
```

Comment:
On the other hand, in your comment in the swift interface you say that the key can be a passphrase like "boto" rather than a real random string. If we're going to count on it being a real random string we should remove that earlier comment.